spec/services/access_request_approval_service_spec.rb

```diff
@@ -130,5 +130,11 @@
         end
       end
     end
+
+    context "writing data concurrently" do
+      it "writes to both user_permission and organisation_user" do
+        expect { call_service! }.to change { OrganisationUser.count }.by(1).and change { UserPermission.count }.by(access_request.requester.organisations.flat_map(&:providers).count)
```

Comment:
could we break this line up? or `access_request.requester.organisations.flat_map(&:providers).count` could go into a variable and assert against that, might read a little better